core/src/test/java/io/crate/monitor/SysInfoUtilTest.java

```diff
@@ -117,6 +121,40 @@ public void testParseRedHatVendor() throws IOException {
         assertThat(util.info().vendorCodeName(), is("Santiago"));
     }
 
+    @Test
+    public void testGenericVendor() throws IOException {
```

Comment:
Could be renamed to something like `testVendorVersionCanBeParsedFromDifferentDefinitions` - ideally we'd have 1 test per input (or use junit params)

Reply:
I was about to change that already